app/ui_components.R

```diff
@@ -7,7 +7,7 @@ help_html <- helpText(
         <br>
         <p>
             <b>Discussion/Questions</b><br>
-            <a href="https://fhdata.slack.com/archives/CJFP1NYSZ"><span class="badge bg-secondary"><i class="fa-brands fa-slack"></i> Slack</span></a> #question-and-answer channel
+            <a href="https://fhdata.slack.com/archives/CJFP1NYSZ"><span class="badge bg-secondary"><i class="fa-brands fa-slack"></i> Slack</span></a> #workflow-managers channel
```

Comment:
In the PROOF call we changed the channel to `#question-and-answer`. You want to change it back?

Reply:
No strong feelings about this. :)

Reply:
There is already a specific channel for workflow managers in FH Data Slack and historically folks using WDL and Cromwell have discussed challenges there.  Either works but it would be easier for us to know we need to respond if it's in the workflow manager channel not in question and answer.  but the group wants it in question-and-answer and are willing to CONSISTENTLY respond to questions there promptly, I don't really care.

Reply:
okay i think workfow manager is good